pkg/manager/setconfig.go

```diff
@@ -27,85 +27,80 @@ import (
 // SetConfigAlreadyApplied is a string constant for "Already applied:"
 const SetConfigAlreadyApplied = "Already applied:"
 
-// SetNetworkConfig sets the given value, according to the path on the configuration for the specified target
-func (m *Manager) SetNetworkConfig(configName store.ConfigName, updates map[string]*change.TypedValue,
-	deletes []string) (change.ID, store.ConfigName, error) {
-
-	// Look for an exact match or then a partial match
-	deviceConfig, ok := m.ConfigStore.Store[configName]
-	if !ok {
-		similarDevices := make([]store.ConfigName, 0)
-		for key := range m.ConfigStore.Store {
-			deviceNameFromKey := string(key)[:strings.LastIndex(string(key), "-")]
-			if deviceNameFromKey == string(configName) {
-				similarDevices = append(similarDevices, key)
-			}
-		}
+// ValidateNetworkConfig validates the given updates and deletes, according to the path on the configuration
+// for the specified target
+func (m *Manager) ValidateNetworkConfig(configName store.ConfigName, updates map[string]*change.TypedValue,
+	deletes []string) (change.ID, error) {
```

Comment:
now that you don't store it I think there's no point at all in returning this change.ID

Reply:
done